### DISCUSSION TITLE
how  can i  label add a color

### DISCUSSION THREAD

**Discussion by oz-hjx210** [filamentphp/filament#6981] [Help] [answered]
Forms\Components\TextInput::make('titletw')->label('<span class="red">名稱(tw):有填前臺顯示</span>')
it can't work 
is an other type  to add a color in label

**Comment by pboivin** [comment, marked as answer]
Maybe this can help:
https://filamentphp.com/tricks/add-a-link-or-html-to-field-label

**Reply by oz-hjx210** [reply]
Thank you very much, this is what I want.
